stripe/res/values/strings.xml

```diff
@@ -1,2 +1,5 @@
 <resources>
+    <string name="card_number_hint">1234 5678 9012 3456</string>
```

Comment:
Flagging that this number doesn't pass luhn validation. If you're looking for a "valid" test number, we frequently use `4242 4242 4242 4242`

https://stripe.com/docs/testing